string locomotion_mode
---
string response